tests/NotLoggedIn/Auth/register.spec.js

```diff
@@ -0,0 +1,61 @@
+import { test } from '../../_fixtures/fixtures';
+import * as allure from 'allure-js-commons';
+import { expect, testStep } from '../../../src/common/helpers/pwHelpers.js';
+import { AuthPage } from '../../../src/ui/pages/AuthPage.js';
+
+test('User can register (positive)', async ({ page }) => {
+  allure.severity?.('critical');
+
+  const username = `pb_${Date.now()}`;
+  const password = 'Test1234!';
+
+  const auth = new AuthPage(page);
+
+  await auth.openHome();
+
+  await testStep('Open Register form', async () => {
+    await auth.openRegister();
+    await expect(page.getByRole('heading', { name: 'Signing up is easy!' })).toBeVisible();
+  });
+
+  await testStep('Fill and submit registration', async () => {
+    await page.fill('#customer\\.firstName', 'John');
```

Comment:
1. create a methods for this test in page object
2. use faker instead of hardcoding